docker-compose.yml

```diff
@@ -13,14 +13,14 @@ services:
     # tty: true
     depends_on:
       - server
-    command: ["sh", "-c", "wait-for-server && cd-bootstrap-remote"]
+    command: ["sh", "-c", "./bin/wait-for-server.sh && ./bin/cd-bootstrap-remote.sh"]
```

Comment:
I don't know the root cause, but I tried on Windows, I have to add the directory path.